### PR TITLE
VCC2020 database

### PR DESCRIPTION
**Pull request by unilight**
The organizers of VCC2020 released the database, so I am modifying the recipes to automatically download the database. Also, since we have the ground truth files and the transcriptions of the evaluation files, we can calculate objective results like MCD, CER, and WER.

Also, I changed `utils/mcd_calculate.py` to detect voiced/unvoiced segments by power rather than f0, in order to eliminate the influence of f0 extraction errors.

- [x] vc1_task1
- [x] vc1_task2
- [x] voc1
- [x] update READMEs

### CODE REVIEW

**Review by kan-bayashi — APPROVED** [review]
Sorry for the late review.
LGTM.
If you can modify the numpy style docstring to google’s one, that is great.

**Comment by unilight** [comment]
> Sorry for the late review.
> LGTM.
> If you can modify the numpy style docstring to google’s one, that is great.

Hmm.. I didn't touch python scrips though... or do you mean other files as well?

File: utils/mcd_calculate.py [hunk 1]
```diff
@@ -65,6 +65,90 @@ def low_cut_filter(x, fs, cutoff=70):
     return lcf_x
 
 
+def spc2npow(spectrogram):
+    """Calculate normalized power sequence from spectrogram
+
+    Parameters
```
Comment (kan-bayashi) [inline utils/mcd_calculate.py:71]: I mean this part.

**Comment by kan-bayashi** [comment]
Maybe you copied from sprocket?
Then, let us keep as is.

**Pull request merged**